### PR TITLE
docs: list all the support options

### PR DESCRIPTION
**Pull request by zimbatm**
Part of #388

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -39,7 +39,17 @@ If you see a project that is under-maintained, submit PRs to fix it as if it was
 
 If you need to contact the nix-community admins, they are listed [here](https://github.com/nix-community/infra#nix-community-administrators).
 
-Come chat with us in [#nix-community](https://matrix.to/#/#nix-community:nixos.org) using the matrix protocol!
+## Support
+
```
Comment (zowoq) [inline docs/README.md:43]: Could we just have a link to https://github.com/nix-community/infra#support here like we do for the admin list above?

Comment (zimbatm) [reply docs/README.md:43]: I thought about it but it adds friction to people coming on the website. We probably want to consolidate the site, README and also the .github/README at some point.

Comment (zowoq) [reply docs/README.md:43]: > I thought about it but it adds friction to people coming on the website.

If they want a list of admins they need to click a link anyway so I don't think it's any worse.

File: docs/README.md [hunk 2]
```diff
@@ -39,7 +39,17 @@ If you see a project that is under-maintained, submit PRs to fix it as if it was
...
+For casual chat, come join us in the [nix-community](https://matrix.to/#/#nix-community:nixos.org) room.
+
+For urgent issues, ping us in the Matrix room (see the admin list below).
```
Comment (zowoq) [inline docs/README.md:46]: To keep these sections the same probably easier to move support above admins.

File: docs/README.md (hunk 1 above)
Comment (zimbatm) [reply docs/README.md:39]: This is only true for 1/4 of contact methods. It's important that visitors see that info at a first glance and not have to go dig for it.

But also, we probably want to refactor all of this. I will merge this as it's already an improvement and keep iterating on the docs.

File: docs/README.md (hunk 2 above)
Comment (zimbatm) [reply docs/README.md:42]: thanks

**Comment by zimbatm** [comment]
bors merge

**Pull request merged**